lib/fastlane/actions/swiftlint.rb

```diff
@@ -5,7 +5,21 @@ def self.run(params)
         if `which swiftlint`.to_s.length == 0 and !Helper.test?
           raise "You have to install swiftlint using `brew install swiftlint`".red
         end
-        Actions.sh("swiftlint")
+
+        command_prefix = [
+          'cd',
+          File.expand_path('.').shellescape,
+          '&&'
+        ].join(' ')
+        swiftlint_args = "> #{params[:report_file]}"
```

Comment:
given that this is a new option, i think that we should make it optional. I'm definitely open to changing that opinion, but i if people already have the swiftlint action in their work flow, making this non-optional would break them right?